cmd/ctr/commands/images/crypt_utils.go

```diff
@@ -250,14 +252,22 @@ func cryptImage(client *containerd.Client, ctx gocontext.Context, name, newName
 		modified bool
 		newSpec  ocispec.Descriptor
 	)
+
+	ls := client.LeasesService()
+	l, err := ls.Create(ctx, leases.WithRandomID(), leases.WithExpiration(5*time.Minute))
+	if err != nil {
+		return images.Image{}, err
+	}
+
 	if encrypt {
-		newSpec, modified, err = images.EncryptImage(ctx, client.ContentStore(), image.Target, cc, lf)
+		newSpec, modified, err = images.EncryptImage(ctx, client.ContentStore(), ls, l, image.Target, cc, lf)
 	} else {
-		newSpec, modified, err = images.DecryptImage(ctx, client.ContentStore(), image.Target, cc, lf)
+		newSpec, modified, err = images.DecryptImage(ctx, client.ContentStore(), ls, l, image.Target, cc, lf)
 	}
 	if err != nil {
 		return image, err
 	}
+	defer ls.Delete(ctx, l, leases.SynchronousDelete)
```

Comment:
This should go further up right after the `ls.Create()`

Reply:
Yup - will fix this! Thanks.